cms/tests/test_rendering.py

```diff
@@ -258,7 +258,7 @@ def test_processors(self):
         from djangocms_text_ckeditor.cms_plugins import TextPlugin
         from cms.plugin_pool import plugin_pool
 
-        instance = CMSPlugin.objects.all()[0].get_plugin_instance()[0]
+        instance = CMSPlugin.objects.order_by('pk')[0].get_plugin_instance()[0]
```

Comment:
@czpython This problem is strange. Only in MySQL 5.6 when you do `model.objects.all()` then order when printing this(`print(qs)`) is correct, then you do `model.objects.all()[0]` or `model.objects.first()` and it retrieve last element, not first from the list. In MySQL 5.7, postgres, sqlite it's all good. Django version 1.11.14, mysqlclient 1.3.13. Root of the problem is that this query (executed by __getitem__ and set_limits on QuerySet):
```sql
'SELECT `cms_cmsplugin`.`id`, `cms_cmsplugin`.`placeholder_id`, `cms_cmsplugin`.`parent_id`, `cms_cmsplugin`.`position`, `cms_cmsplugin`.`language`, `cms_cmsplugin`.`plugin_type`, `cms_cmsplugin`.`creation_date`, `cms_cmsplugin`.`changed_date` 
FROM `cms_cmsplugin` 
ORDER BY `cms_cmsplugin`.`position` ASC LIMIT 1'
```
resulting in last item instead of first on the list, when positions are the same (in this case `1`).

Reply:
Replace this with self.test_placeholders['main'].get_plugins('en').first()